### PR TITLE
[TICKET 0]:wrench: updates 

### PR DESCRIPTION
**Pull request by czhoffmann23**
## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?
Current template need some structure, ui, fonts and flows updates
## What is the new behavior?
1.  Create new structure
1.1 Layout: manage all the structure in the application (ex: navbar and childrens). ⚡ 🔧 
1.2 Screens: manage all the components in pages (headers, cards, footers, etc). ⚡ 🔧  
1.3 Pages: manage all the main routes. ⚡ 🔧 
1.4 Services: manage all the connections (queries, mutations, etc). ⚡ 🔧 
1.5 Theme: manage theme according to chakra ui structure. ⚡ 🔧 
1.6 Components: manage global components ⚡ 🔧 
2. Add React query structure 🔧 
3. Update UI 🎨
4. Update ReactJS ⚡ 🚀 
5. Add some svg examples 🎨 
6. Updates in environment file example ✏️ 
7. Updates in husky commands to manage `yarn build` in git push trigger 
7.1 this going to prevent preview vercel errors 💀 
## How can your PR break the app?
Its everything new
## Screenshots (optional)
**DESKTOP** 💻 

image
image
image

**MOBILE** 📱 

image
image
image

### CODE REVIEW

**Comment by tomcastro** [comment]
👀

**Review by Yhozen — COMMENTED** [review]
:money_mouth_face:

**Comment by Byevenes** [comment]
Boooooof ❤️‍🔥

File: src/screens/home/RenderCards.tsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+import { STACK } from './constant'
+
+type StackItem = {
+  label: string
+  url: string
+  logo: number
+}
+
+export const RenderCards = () => {
```
Comment (Byevenes) [inline src/screens/home/RenderCards.tsx:14]: Quizás cambiar el switch ya que todos serán como componentes a mostrar diferentes a esta tipo y usar un useMemo para el getComponent para el resultado, pd: lo saque de una conversa que tuvimos del tema con @Yhozen 🦀 
 
image

**Pull request merged**